tests/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you rename this to `.eslintrc.js` and `module.export` this?  Using JS files will allow us to cascade rules a bit easier...